frb_codegen/src/ir/field.rs

```diff
@@ -4,5 +4,6 @@ use crate::ir::*;
 pub struct IrField {
     pub ty: IrType,
     pub name: IrIdent,
+    pub non_final: bool,
```

Comment:
Shall we name it `final` instead of `non_final` to avoid double negation :)

Reply:
final is a reserved keyword in Rust. This is why I stuck with non_final.

Reply:
what about `is_final`?

Reply:
Ok, I'll change it.